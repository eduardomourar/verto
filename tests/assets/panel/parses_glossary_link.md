{panel type="teacher-note" summary="GlossaryLink"}
The present searching game in this section is split into two parts, the first corresponds to the Linear Search algorithm (also known as Sequential Search) and the second corresponds to {glossary-link term="Binary Search"}Binary Search{glossary-link end}. We recommend you play through the levels yourself for a while, or after reading this description. It is based on the [CS Unplugged Battleships game](http://csunplugged.com/searching-algorithms) which can be used as a classroom activity to enforce the lessons in this chapter (the hashing activity is not used for the present searching game).
{panel end}