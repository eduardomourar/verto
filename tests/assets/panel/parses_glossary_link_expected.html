<div class="clearfix">
<ul class="collapsible panel" data-collapsible="accordion">
<li class="panel-selector panel-teacher-note">
<div class="collapsible-header">
<div class="panel-heading">Teacher Note: GlossaryLink</div>
<div class="dropdown-menu-arrow">â–¼</div>
</div>
<div class="collapsible-body">
<p>
The present searching game in this section is split into two parts, the first corresponds to the Linear Search algorithm (also known as Sequential Search) and the second corresponds to <a href='../further-information/glossary.html#Binary Search' id='glossary-0' class='glossary-anchor-link glossary-link-back-reference'>Binary Search</a>. We recommend you play through the levels yourself for a while, or after reading this description. It is based on the [CS Unplugged Battleships game](http://csunplugged.com/searching-algorithms) which can be used as a classroom activity to enforce the lessons in this chapter (the hashing activity is not used for the present searching game).
</p>
</div>
</li>
</ul>
</div>
